src/components/notifications/AppNotifications/AppNotificationDropdown/index.tsx

```diff
@@ -23,6 +25,10 @@ const AppNotificationDropdown: React.FC<IAppNotificationDropdownProps> = ({
 }) => {
   const { pushClientProxy } = useContext(W3iContext)
 
+  const handleMute = useCallback(() => {
```

Comment:
I don't think we have such capability yet in Notify client. cc @bkrem @devceline

Reply:
We don't. Best to hide this button 👍